### PR TITLE
Dont inject popups into feeds

### PR DESCRIPTION
**Pull request by claudiulodro**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR fixes an issue where popup content was getting injected into the feeds, so the first line of each item in a site's RSS feed would be the popup content.

### How to test the changes in this Pull Request:

1. Before applying this patch, create a popup:
Screen Shot 2020-02-12 at 2 21 01 PM

2. Visit `/feed` on your site. Observe the pop up content is in the description for each item:
Screen Shot 2020-02-12 at 2 21 10 PM

3. Apply this patch. Refresh. Observe the pop up content is not in the description for each item:
Screen Shot 2020-02-12 at 2 22 44 PM

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: includes/class-newspack-popups-inserter.php [hunk 1]
```diff
@@ -70,7 +70,7 @@ public function __construct() {
 	 * @return string The content with popup inserted.
 	 */
 	public static function popup( $content = '' ) {
-		if ( is_admin() ) {
+		if ( is_admin() || is_feed() ) {
```
Comment (philipjohn) [inline includes/class-newspack-popups-inserter.php]: Is this hook only supposed to run for popups? If so, I'd recommend making this conditional check that the current post is of the right post type. E.g.:

```
if ( Newspack_Popups::NEWSPACK_PLUGINS_CPT !== get_post_type() ) {
    return;
}
```

Comment (claudiulodro) [reply includes/class-newspack-popups-inserter.php]: No, it injects the popup into post content.

**Comment by claudiulodro** [comment]
Let me add a couple tweaks to prevent popups showing on all excerpts too before reviewing.

**Comment by claudiulodro** [comment]
I've changed the logic a little so it will prevent popups from getting injected into excerpts on feeds and archives.

**Pull request merged**